go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/nwidger/jsoncolor v0.3.2
 	github.com/oklog/ulid/v2 v2.1.0
 	github.com/openfga/api/proto v0.0.0-20240529190005-394820dfc109
-	github.com/openfga/go-sdk v0.4.0
```

Comment:
**Note:** Pinning to appropriate commit. Will defer the decision of wether to wait for a tagged release or not.

Reply:
@willvedd we pushed out v0.5.0 of the go SDK so we can pull that in now

Reply:
✅